Eventually becoming a full pass (start at the top) (#63)
datalad/datalad-paper-joss: paper.md

   orcid: 0000-0003-2917-3450
   affiliation: 2
 - name: Jason Gors  # 60 commits, issues: opened 2 participated in 1

>    affiliation: FIXME

Reviewer: Is having affiliation strictly required? Would be very non inclusive from joss to forbid citizen scientists from publishing

Author: That is not my concern. My concern is that without an affiliation declaration they all get `1`, which is Dartmouth College ATM. If that is OK, or even intentional, I suggest to make that explicit, rather than rely on that implicit behavior.

Screenshot without FIXME:

![image](https://user-images.githubusercontent.com/136479/115115027-744a4c80-9f92-11eb-993e-e9c6d7fe6a1c.png)

Reviewer: ah, nicely spotted -- I have missed that and it was not intentional.

However, data have remained a “2nd-class citizen” in the contemporary scientific process, despite the increasing embracement of the FAIR principles [@FAIR2016].
1) Disconnected data portals force a cacophony of data access and authentication methods upon the scientific community; 2) data versioning is rarely performed, or nowhere close to the precision and inspectability that is standard in software development; and 3) data provenance---if shared at all---is often not complete, because their capture is rarely considered to be an integral part of the scientific process.
DataLad aims to solve all these issues by providing targeted interfaces and interoperability adapters to standard tools and services, scientific and commercial.
It enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
In order to maximize its utility and target audience, DataLad is available for all major operating systems, and can be integrated into established workflows and environments with minimal adjustments.

## Why Git and git-annex?

> Git is the standard version control system for software development[^1].

Reviewer: there is no standard for designing version control systems, and "**the** standard" really throws me off as hinting on it to be the only one. What about simpler

```suggestion
Git is the most popular version control system for software development[^1].
```

It is a distributed content management system, specifically tuned towards managing and collaborating on text files, and excels at making all content committed to Git reliably and efficiently available to all clones of a repository.
At the same time, Git is not designed to efficiently handle large (e.g., over a gigabyte) or binary files [see, e.g., @opensource:git-binary].
This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files, or even to fully remove file content without severely impacting the integrity of a repository.
Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.

> File content managed by git-annex is placed into a managed repository annex, instead of committing it directly
> to Git.

Reviewer: Just for consistency in formatting
```suggestion
File content managed by git-annex is placed into a managed repository annex, instead of committing it directly to Git.
```

> Instead of the file content, git-annex only commits a compact reference that enables identification and association of file name and content.

Reviewer: ```suggestion
Instead of the file content, git-annex only commits a compact reference that enables identification and association of a file name with the content.
```
since it is not necessarily a reversible relationship, and content might be associated with multiple files

File content references are typically based on a checksum of the content.
Using these identifiers, git-annex tracks content availability across all repository clones (local or remote), or external resources such as URLs pointing to individual files on the web.
Upon user request, git-annex automatically manages data transport to and from a local repository annex at a granularity of individual files.
With this simple approach, git-annex enables separate and optimized implementations for identification and transport of arbitrarily large files, using an extensible set of protocols, while retaining the distributed nature and compatibility with versatile workflows for versioning and collaboration provided by Git.

[^1]: https://en.wikipedia.org/wiki/Git#Adoption

> ## Why does DataLad add to Git and git-annex?

Reviewer: odd if not incorrectly formed sentence breaking alignment with paragraph titles ATM (but might be appropriate for #64 ). I will for now revert to the original

```suggestion
## Why Git and git-annex alone are not enough?
```
